fix(biome_css_parser): fix css parser cases (#4374)
biomejs/biome: CHANGELOG.md

#### Bug fixes

> - Fix CSS parser case error, if `@-moz-document url-prefix(https://example.com)` or `@-moz-document domain(example.com)` should be valid. Contributed by @eryue0220
> 

Reviewer: ```suggestion
- Fix CSS parser case error, `@-moz-document url-prefix(https://example.com)` and `@-moz-document domain(example.com)` are now valid. Contributed by @eryue0220

```
